keyboards/smartkit/macro8/keymaps/default/rules.mk

```diff
@@ -0,0 +1,14 @@
+ENCODER_MAP_ENABLE = yes
+VIA_ENABLE = yes
+#ENCODER_ENABLE = yes
+#ENCODER_MAP_ENABLE = yes
+
+
+#LEADER_ENABLE = no
+#FAUXCLICKY_ENABLE = no
+#MIDI_ENABLE = no
+#BLUETOOTH_ENABLE = no
+#KEY_LOCK_ENABLE = no
+#TERMINAL_ENABLE = no
+
+#RGBLIGHT_ENABLE=yes
```

Comment:
```suggestion
```
VIA should not be enabled on the default keymap. The rest of these can simply go.